src/inet/InetInterface.cpp

```diff
@@ -574,7 +575,7 @@ static struct if_nameindex * backport_if_nameindex(void)
         }
     }
 
-    retval = (struct if_nameindex *) malloc((maxIntfNum + 1) * sizeof(struct if_nameindex));
+    retval = (struct if_nameindex *) MemoryAlloc((maxIntfNum + 1) * sizeof(struct if_nameindex));
```

Comment:
Use `MemoryFree(intf->if_name)` instead of `free(intf->if_name)` in line 591 below.

Reply:
This is allocated by  tmpval[intfIter].if_name  = strdup(addrIter->ifa_name);